scripts/README.md

```diff
@@ -0,0 +1,122 @@
+# Edge Cluster Multi-Layer Setup using Bash Scripts  
+
+
+
+### Virtual Machine Configuration 
+
+
+
+•	**3 Ubuntu 18.04 VMs, one for cloud-core, two for edge-core.**   
+•	Open the port of 10000 and 10002 in the security group of the cloud-core machine and edge-core machine   
+•	16 GB RAM, 16 vCPUs, 128 GB storage.    
+
+####     Machine 1: Cloud Core Node 
+####     Machine 2: Edge Node with Control Plane 
+####     Machine 3: Edge Worker Node
+
+
+## Steps to configure 'sshd' before running the scripts in all the three machines:
+
+**Switch to ROOT user:**
+        
+        sudo -i
+        
+**Edit the 'sshd_config' file:**
+
+       vi /etc/ssh/sshd_config
+       
+       
+**Here modify line no. 32 and line no. 56 by uncommenting and updating to `PermitRootLogin yes` and `PasswordAuthentication yes`**
+
+
+
+   ![image](https://user-images.githubusercontent.com/95343388/152476470-8fb9d893-23bb-4666-84fc-7996f6d132a7.png)
+   
+   
+   
+
+**Now reload the sshd service:**
+     
+     
+       systemctl reload sshd
+       
+       
+**Set the ROOT password of the Machine**
+
+
+       passwd root
```

Comment:
if user can change into root user with "sudo su", do we really *need* to set password for root? for security reason.